pynxtools/dataconverter/readers/ellips/reader.py

```diff
@@ -439,9 +440,9 @@ def read(
                 # using a proper unit parsing logic
                 template[f"/ENTRY[entry]/plot/DATA[{key}]/@units"] = "degree"
                 if dindx == 0 and index == 0:
-                    template[
-                        f"/ENTRY[entry]/plot/DATA[{key}]/@long_name"
-                    ] = f"{plot_name} (degree)"
+                    template[f"/ENTRY[entry]/plot/DATA[{key}]/@long_name"] = (
```

Comment:
Please revert this formatting change. It's confusing because nothing has chanced here and it has nothing to do with this PR

Reply:
No, ruff auto formatting